UnityProject/Assets/Scripts/Tool/DoorWedger.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DoorWedger : MonoBehaviour, IInteractable<HandApply>, ICheckedInteractable<HandApply>
+{
+    [SerializeField] [Tooltip("The time in seconds it takes to wedge open a door with this tool.")]
+    private float openingDelay = 10f;
+
+    public bool WillInteract(HandApply interaction, NetworkSide side)
+    {
+        if (!DefaultWillInteract.Default(interaction, side)) return false;
+
+        if (Validations.IsTarget(gameObject, interaction)) return false;
+        return true;
+    }
+    
+    public void ServerPerformInteraction(HandApply interaction)
+    {
+        var targetObject = interaction.TargetObject.GetComponent<DoorController>();
+
+        if (targetObject == null)
+        {
+            Chat.AddExamineMsgFromServer(interaction.Performer, "DEBUG: You hit NULL.");
```

Comment:
I belive this shouldn't be here.

Reply:
Good catch, I'm going to update it tomorrow.

Reply:
I just removed this line of code!